src/main/java/com/nike/cerberus/endpoints/authentication/TotpMfaCheck.java

```diff
@@ -34,12 +34,12 @@
 /**
  * Endpoint for verifying the token from the user's MFA device.  Returns the full auth response if verified.
  */
-public class MfaCheck extends StandardEndpoint<MfaCheckRequest, AuthResponse> {
+public class TotpMfaCheck extends StandardEndpoint<MfaCheckRequest, AuthResponse> {
```

Comment:
If this class / endpoint is going to be used for sms and push then a name like `codeHandlingMfaCheck` or something along the that lines, that informs people that it is used for more than just TOTPs.